app/javascript/elm/tests/Data/SelectedSessionTests.elm

```diff
@@ -1,9 +1,7 @@
 module Data.SelectedSessionTests exposing (suite)
 
-import Data.SelectedSession exposing (sensorNameFromId)
+import Data.SelectedSession exposing (..)
```

Comment:
why do you prefer to do this?

Reply:
Don't know. I'll refactor to use an explicit import